src/domain/report-data.ts

```diff
@@ -28,7 +28,7 @@ export class ReportData {
   asHash(): string {
     return crypto
       .createHash('sha256')
-      .update(JSON.stringify(this))
+      .update(JSON.stringify(this.data))
```

Comment:
@tomwwinter, I think the hash should indicate if the actual data (without metadata) is the same? Or do we need a different property for that?

Reply:
Yes. I already changed that my current working branch :)